### ISSUE TITLE
--write_to_movie not working without -pql for FullScreenQuad

### ISSUE THREAD

**Issue by cryvosh** [ManimCommunity/manim#1902]
## Description of bug / unexpected behavior

I'm trying to run [this ](https://gist.github.com/naveen521kk/ea049dc724fa46ba11945069a2f171e3)example (with line 851 removed to satisfy the current manim version).

Running `manim --renderer=opengl cybertruck.py InlineFullScreenQuad --write_to_movie` does not output a movie unless I also add `-pql`.

Without `-pql` it says `Output file #0 does not contain any stream` and `File ready at 'C:\Users\...\InlineFullScreenQuad.mp4'` despite there being no such file at that address.

**Comment by ryanmccauley211** [comment]
Hmm it seemed to work for me, I ran `python -m manim --renderer=opengl ../scenes/cybertruck.py InlineFullScreenQuad --write_to_movie`

and got the output

```
Manim Community v0.8.0

[08/13/21 19:06:41] INFO     Animation 0 : Partial      scene_file_writer.py:415
                             movie file written in '/Us                         
                             ers/ryan/PycharmProjects/m                         
                             anim-community/manim/media                         
                             /videos/cybertruck/1080p60                         
                             /partial_movie_files/Inlin                         
                             eFullScreenQuad/1491410813                         
                             _76435013_522099545.mp4'                           
[08/13/21 19:06:42] INFO                                scene_file_writer.py:597
                             File ready at '/Users/ryan                         
                             /PycharmProjects/manim-com                         
                             munity/manim/media/videos/                         
                             cybertruck/1080p60/InlineF                         
                             ullScreenQuad.mp4'                                 
                                                                                
                    INFO     Rendered InlineFullScreenQuad          scene.py:234
                             Played 1 animations                       
```

The output video also worked. Are there anymore output logs you can share by running with `-v DEBUG`. If you can also share output of `ffmpeg -version` it might also be useful

**Comment by Kiran-Raj-Dev** [comment]
This seems to have been fixed at some point in `v0.15.0`.
Also closing due to inactivity.
